src/templates/smfcfg.yaml.j2

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: 2022-present Intel Corporation
+# SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
+# Copyright 2019 free5GC.org
+#
+# SPDX-License-Identifier: Apache-2.0
+
+info:
+  version: 1.0.0
+  description: SMF initial local configuration
+
+configuration:
+  smfDBName: {{ smf_database_name }}
+  enableDBStore: false
+  enableUPFAdapter: true
+  debugProfilePort: 5001
+  mongodb:
+    name: {{ default_database_name }}
+    url: {{ default_database_url }}
+  kafkaInfo:
+    brokerUri: "sd-core-kafka-headless"
+    brokerPort: 9092
+    topicName: "sdcore-data-source-smf"
```

Comment:
@gruyaume suggested to remove the `kafka` config in the [amf pr](https://github.com/canonical/sdcore-amf-operator/pull/1).
Should we do the same here?